packages/graph/clients/typescript/api.ts

```diff
@@ -635,7 +634,7 @@ export interface EntityStructuralQuery {
    * @type {UnresolvedTimeProjection}
    * @memberof EntityStructuralQuery
    */
-  timeProjection?: UnresolvedTimeProjection;
+  timeProjection: UnresolvedTimeProjection;
```

Comment:
We talked offline about changing the `UnresolvedTimeProjection` type to make the undefined axis timestamp value a `null` value instead. Just writing it here as well.
We could do this by making the `Timestamp` object nullable, but that really isn't ideal. Perhaps the other solution is to somehow make use of an enum for either the `Timestamp` object or null